src/stages/normalize/patchers/FunctionApplicationPatcher.js

```diff
@@ -82,7 +82,11 @@ export default class FunctionApplicationPatcher extends NodePatcher {
     let { args } = this;
     let lastArg = args[args.length - 1];
     if (lastArg.isMultiline()) {
-      this.insert(this.contentEnd, `\n${this.getIndent()})`);
+      if (this.slice(this.contentEnd, this.contentEnd + 2).match(/\n\s*\./)) {
```

Comment:
The `+ 2` and the use of regexes seems a bit fragile. Maybe you could use one of the various `NodePatcher` to  methods to find a DOT token after the end of the expression?